xrmocap/model/registrant/smplifyx.py

```diff
@@ -36,11 +40,24 @@ def __set_keypoint_indexes__(self) -> None:
         self.body_keypoint_idxs = get_keypoint_idxs_by_part(
             'body', convention=convention)
 
+        self.shoulder_keypoint_idxs = get_keypoint_idxs_by_part(
+            'shoulder', convention=convention)
+
+        self.hip_keypoint_idxs = get_keypoint_idxs_by_part(
+            'hip', convention=convention)
+
+        self.foot_keypoint_idxs = get_keypoint_idxs_by_part(
+            'foot', convention=convention)
+
     def get_keypoint_weight(self,
                             use_shoulder_hip_only: bool = False,
```

Comment:
If we've defined shoudler_weight and hip_weight, why don't we just use them and abondon `use_shoulder_hip_only`?
I suggest that we remove `use_shoulder_hip_only` from args list, and:
```python
if 'use_shoulder_hip_only' in kwargs:
    other_weights = 0.0
    self.logger.warning('use_shoulder_hip_only is deprecated, please ...')
```

Reply:
For 0.8.0, we use warning. For 0.9.0 this feature can be removed.

Reply:
When we set  `use_shoulder_hip_only=True`,  we make sure all other weights will be set to 0.0 except shoulder and hip weights. When `use_shoulder_hip_only=False`,  the default weight is 1.0 for all keypoints unless parts' weights are specified. If we remove this feature, users need to set every part's weight to 0.0 by themselves and there is a risk that some weights are not turned on/off properly.

Reply:
Keypoints weights are manually controlled in [7c8dd4c](https://github.com/openxrlab/xrmocap/pull/52/commits/7c8dd4c339aed2d2666a708df69fbfd43c1488db). To avoid spam, `logger.warning` is not used, instead, it will return an error if weights setting conflicts with `use_shoulder_hip_only` for now. For 0.9.0 `use_shoulder_hip_only` can be removed.